packages/core/src/modules/event/event-company-repository.ts

```diff
@@ -64,6 +63,6 @@ export class EventCompanyRepositoryImpl implements EventCompanyRepository {
       query = query.orderBy("id", "desc")
     }
     const events = await query.execute()
-    return events.map(mapToEvent)
+    return events.map((e) => EventSchema.parse(e))
```

Comment:
Can we not do mapToEvent here? alternatively mapToEventWithCommittees?

Reply:
Fixed